packages/fx-core/src/component/driver/apiKey/interface/updateApiKeyArgs.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export interface UpdateApiKeyArgs {
+  registrationId: string; // The registration id of the api key
+  name: string; // The name of Api Secret
+  appId: string; // Teams app id
+  apiSpecPath: string; // The location of api spec file
+  applicableToApps?: string; // What app can access the api key. Values can be "SpecificApp" or "AnyApp". Default is "AnyApp".
```

Comment:
Which app can access the API key? Values can be "SpecificApp" or "AnyApp". Default is "AnyApp".

Reply:
Fixed, thanks!